streamwebs_frontend/streamwebs/tests/views/test_create_site_view.py

```diff
@@ -19,8 +19,6 @@ def test_view_with_bad_blank_data(self):
 
         self.assertFormError(response, 'site_form', 'site_name',
                              'This field is required.')
-        self.assertFormError(response, 'site_form', 'location',
```

Comment:
This is a valid error if no coordinates are entered, though the message should be a little more informative, like "you must select a location". Let's make sure we are testing correctly for the case where coordinates are not entered vs the case where a name is not entered, and that the code does the right thing.